runbotics/runbotics-orchestrator-ui/src/src-app/views/users/UsersListView/UsersListEdit/UsersListEdit.types.ts

```diff
@@ -0,0 +1,25 @@
+import { IUser } from 'runbotics-common';
+
+export interface UsersListEditDialogProps {
+    open: boolean;
+    openDeleteTab: () => void;
```

Comment:
delete tab? whats that?

Reply:
I think I should rename it to openDeleteDialog, it will be more clear.